### PR TITLE
Create Token Constants endpoint

### LINKED ISSUE #1441: Create Token Constants endpoint
### Description

Create the following endpoint.
```
HTTP GET /api/v3/token/constants
RPC  get.token.constants
```

Update the response for the following endpoint
```
HTTP GET /api/v3/pos/constants
RPC  get.pos.constants
```

### Acceptance Criteria

- The endpoints listed above are created/updated
- The response is matching as described below
- All the methods, variables, files are named appropriately
- Swagger is up-to-date
- Integration tests are up-to-date

### Additional information
Relevant SDK endpoints:
- `token_getInitializationFees`
- `pos_getRegistrationFee`

*Response: /token/constants*

200 OK
```jsonc
{
    "data": {
        "extraCommandFees": {
            "userAccountInitializationFee": "5000000",
            "escrowAccountInitializationFee": "5000000",
        }
    },
    "meta": {}
}
```

*Response: /pos/constants*

200 OK
```jsonc
{
    "data": {
        ...
        "extraCommandFees": {
            ...
            "validatorRegistrationFee": "1000000000", <-- add this
        }
    },
    "meta": {}
}
```

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #1441 #1398 

### How was it solved?

- [x] The endpoint listed above is created
  - [x] HTTP GET /api/v3/token/constants
  - [x] RPC  get.token.constants
- [x] Response is updated specified
- [x] All the methods, variables, files are named appropriately
- [x] Swagger is up-to-date
- [x] Integration tests are up-to-date
- [x] Transaction processors are renamed appropriately
- [x] Transaction processors are implemented for all the available commands from Lisk SDK/Core

### How was it tested?
Local

### CODE REVIEW

File: services/gateway/apis/http-version3/swagger/definitions/tokens.json [hunk 1]
```diff
@@ -191,5 +191,46 @@
...
+                        "type": "object",
+                        "required": [
+                            "userAccountInitializationFee",
+                            "escrowAccountInitializationFee"
+                        ],
+                        "properties": {
+                            "userAccountInitializationFee": {
+                                "type": "string",
+                                "example": "5000000",
+                                "description": "The extra fee to be supplied for initialising user account."
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/swagger/definitions/tokens.json:220]:
```suggestion
                                "description": "The extra fee to be supplied for initializing a new user account."
```

File: services/gateway/apis/http-version3/swagger/definitions/tokens.json [hunk 2]
```diff
@@ -191,5 +191,46 @@
...
+                            },
+                            "escrowAccountInitializationFee": {
+                                "type": "string",
+                                "example": "5000000",
+                                "description": "The extra fee to be supplied for initialising escrow account."
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/swagger/definitions/tokens.json:225]:
```suggestion
                                "description": "The extra fee to be supplied for initializing a new escrow account."
```

File: services/gateway/apis/http-version3/methods/modules/token/tokenConstants.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		constantsSchema[this.swaggerApiPath] = { get: {} };
+		constantsSchema[this.swaggerApiPath].get.tags = this.tags;
+		constantsSchema[this.swaggerApiPath].get.summary = 'Requests token Module Constants.';
+		constantsSchema[this.swaggerApiPath].get.description = getSwaggerDescription({
+			rpcMethod: this.rpcMethod,
+			description: 'Requests all the configured constants for the token module.',
+		});
+		constantsSchema[this.swaggerApiPath].get.responses = {
+			200: {
+				description: 'Returns all the configured constants for the token module.',
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/methods/modules/token/tokenConstants.js:37]:
```suggestion
		constantsSchema[this.swaggerApiPath].get.summary = 'Requests Token module constants.';
		constantsSchema[this.swaggerApiPath].get.description = getSwaggerDescription({
			rpcMethod: this.rpcMethod,
			description: 'Requests all the configured constants for the Token module.',
		});
		constantsSchema[this.swaggerApiPath].get.responses = {
			200: {
				description: 'Returns all the configured constants for the Token module.',
```

**Review by sameersubudhi — APPROVED** [review]
Looks good. Just a couple of minor change suggestions.

**Review by sameersubudhi — APPROVED** [review]
Thank you very much for the proactive changes. LGTM 👍

**Review by priojeetpriyom — APPROVED** [review]

**Pull request merged**